packages/bpk-component-horizontal-nav/src/BpkHorizontalNav.js

```diff
@@ -145,29 +153,40 @@ class BpkHorizontalNav extends Component<Props> {
       leadingScrollIndicatorClassName,
       trailingScrollIndicatorClassName,
       showUnderline,
+      type,
       ...rest
     } = this.props;
 
     const classNames = getClassName(
       'bpk-horizontal-nav',
-      showUnderline && 'bpk-horizontal-nav--show-underline',
+      showUnderline &&
+        type === HORIZONTAL_NAV_TYPES.default &&
+        'bpk-horizontal-nav--show-default-underline',
+      showUnderline &&
+        type === HORIZONTAL_NAV_TYPES.light &&
+        'bpk-horizontal-nav--show-light-underline',
       className,
     );
 
-    let children = null;
-    if (!autoScrollToSelected) {
-      children = rawChildren;
-    } else {
+    let children = rawChildren;
+
+    if (autoScrollToSelected || type === HORIZONTAL_NAV_TYPES.light) {
       children = React.Children.map(rawChildren, child => {
-        if (!child || !child.props || !child.props.selected) {
-          return child;
+        const childProps = {};
+
+        if (autoScrollToSelected) {
+          if (child && child.props && child.props.selected) {
+            childProps.ref = ref => {
+              this.selectedItemRef = ref;
+            };
+          }
+        }
+
+        if (type === HORIZONTAL_NAV_TYPES.light) {
+          childProps.type = HORIZONTAL_NAV_TYPES.light;
         }
 
-        return React.cloneElement(child, {
-          ref: ref => {
-            this.selectedItemRef = ref;
-          },
-        });
+        return React.cloneElement(child, childProps);
```

Comment:
Will this blow-up if one of the children is null? I think there was a reason that we were checking that `child` was truthy

Reply:
Ah yeah let me check that

Reply:
I _think_ it should be fine. This is a `children.map` loop so it shouldn't iterate over null values. I'll pop a check in anyway just for safety.